ru.js

```diff
@@ -0,0 +1,777 @@
+module.exports = `

```

Comment:
I’ve checked over all the code, and all should be good now.

Reply:
perfect. thanks!